clarity_ext/extensions.py

```diff
@@ -252,7 +252,6 @@ def run_instance(self, instance):
         extension = instance.__class__
         context = instance.context
         try:
-            context.validate_xml_concordance()
```

Comment:
I don't get this. It was under your council that I implemented a concordance validation for xml's with and without state. Now you fetch artifacts without state as I did, and just remove the validation?

Reply:
It all had to do with that I first refactored the logic. With the refactored logic I was going to put this back in, but then I was confident in that `state` fetches the latest, since it's documented to do that.

I'm happy to put it back in if you think we should